src/App.js

```diff
@@ -1,23 +1,38 @@
 import React from "react";
-import { Button, Form, Row, Col, Card, Input, Typography, message } from "antd";
+import { Button, Form, Row, Col, Card, Input, Typography } from "antd";
 import axios from "axios";
+import { useNavigate } from "react-router-dom";
 
 const Login = () => {
-  const onFinish = (values) => {
-    axios
-      .post("https://dev.sitara.cc/api/smb/user/login", values)
-      .then((res) => {
-        console.log(res.data);
-      })
-      .catch((error) => {
-        console.error(error);
-      });
+  const navigate = useNavigate();
+
+  const onFinish = async (values) => {
+    try {
+      const response = await axios.post(
+        "https://dev.sitara.cc/api/smb/user/login",
+        values
+      );
+      const accesstoken = response?.data?.data?.access_token;
```

Comment:
here instaed of writing res?.data?.data multiple times store it in a variable and use that variable